### PR TITLE
doc(bigtable): declutter landing pages

### PR DESCRIPTION
**Pull request by coryan**
Part of the work for #10157

- - -
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/10545)

### CODE REVIEW

File: google/cloud/bigtable/doc/error-handling.dox [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+@par Example
+@snippet read_snippets.cc read row
+
+@see @ref google::cloud::StatusOr
+@see @ref google::cloud::Status the class used to describe errors.
+@see @ref google::cloud::future for more details on the type returned
+    by asynchronous operations.
+
+*/
```
Comment (dbolduc) [inline google/cloud/bigtable/doc/error-handling.dox:42]: nit: are we missing some sort of EOF?

**Review by dbolduc — APPROVED** [review]

File: google/cloud/bigtable/doc/error-handling.dox (hunk 1 above)
Comment (devbww) [reply google/cloud/bigtable/doc/error-handling.dox:42]: That's "No newline at end of file".

Comment (coryan) [reply google/cloud/bigtable/doc/error-handling.dox]: Fixed

File: google/cloud/bigtable/doc/environment-variables.dox [hunk 2]
```diff
@@ -1,17 +1,50 @@
...
+
+`GOOGLE_CLOUD_CPP_ENABLE_CLOG=yes`: turns on logging in the library. Basically
+the library always "logs" but the logging infrastructure has no backend to
+actually print anything until the application sets a backend or they set this
+environment variable.
+
+@see google::cloud::LogBackend
+@see google::cloud::LogSink
+
+@section pbigtable-env-endpoint Endpoint Overrides
```
[the remark this answers is not in the document]
Comment (coryan) [reply google/cloud/bigtable/doc/environment-variables.dox]: Fixed

File: google/cloud/bigtable/doc/environment-variables.dox [hunk 3]
```diff
@@ -1,17 +1,50 @@
...
 
-@see the [gRPC
-documentation](https://grpc.io/grpc/cpp/md_doc_environment_variables.html) for
-    additional environment variables controlling gRPC.
+`GOOGLE_CLOUD_CPP_ENABLE_TRACING=rpc`: turns on tracing for most gRPC calls.
+The library injects an additional Stub decorator that prints each gRPC
+request and response. Unless you have configured you own logging backend, you
+should also set `GOOGLE_CLOUD_CPP_ENABLE_CLOG` to produce any output on the
+program's console.
+
```
[the remark this answers is not in the document]
Comment (coryan) [reply google/cloud/bigtable/doc/environment-variables.dox:15]: Done

File: google/cloud/bigtable/doc/environment-variables.dox [hunk 4]
```diff
@@ -1,17 +1,50 @@
 /*!
-@page bigtable-environment-variables Environment Variables Used by the Google Cloud Bigtable C++ Client.
+@page bigtable-environment-variables Environment Variable
```
[the remark this answers is not in the document]
Comment (coryan) [reply google/cloud/bigtable/doc/environment-variables.dox]: Fixed

**Pull request merged**